README.md

```diff
@@ -742,7 +740,7 @@ This is a 'catch-all' category for anything that doesn't fit well anywhere else.
 [116]: http://bstring.sourceforge.net/
 [117]: http://troydhanson.github.io/uthash/
 [119]: https://www.enlightenment.org?p=about%252Fefl
-[120]: http://zserge.com/jsmn.html
+[120]: https://zserge.com
```

Comment:
You've missed to include the part after the TLD here. This link lands to a homepage of the website, not the project. Could you please add the `/jsmn.html` to the end of this link?